### PR TITLE
Add variable fixing to GrMhd executable

### PR DESCRIPTION
**Pull request by kidder**
## Proposed changes

Also switch to Fishbone Moncrief analytic solution.

### Types of changes:

- [ ] Bugfix
- [x] New feature

### Component:

- [x] Code
- [ ] Documentation
- [ ] Build system
- [ ] Continuous integration

### Code review checklist

- [ ] The PR passes all checks, including unit tests, `clang-tidy` and `IWYU`. For
  instructions on how to perform the CI checks locally refer to the [Dev guide
  on the Travis CI](https://spectre-code.org/travis_guide.html).
- [ ] The code is documented and the documentation renders correctly. Run `make doc`
  to generate the documentation locally into `BUILD_DIR/docs/html`. Then open
  `index.html`.
- [ ] The code follows the stylistic and code quality guidelines listed in the
  [code review guide](https://spectre-code.org/code_review_guide.html).

### CODE REVIEW

File: src/Evolution/Executables/GrMhd/ValenciaDivClean/EvolveValenciaDivClean.hpp [hunk 1]
```diff
@@ -65,7 +69,8 @@ struct EvolutionMetavars {
   // To switch which analytic solution is evolved you only need to change the
   // line `using analytic_solution = ...;` and include the header file for the
   // solution.
-  using analytic_solution = grmhd::Solutions::SmoothFlow;
+  //  using analytic_solution = grmhd::Solutions::SmoothFlow;
```
Comment (nilsdeppe) [inline src/Evolution/Executables/GrMhd/ValenciaDivClean/EvolveValenciaDivClean.hpp:72]: Is this commented out code intentional?

**Review by nilsdeppe — COMMENTED** [review]
One small question, overall I'm happy with this :+1:

File: src/Evolution/Executables/GrMhd/ValenciaDivClean/EvolveValenciaDivClean.hpp (hunk 1 above)
Comment (kidder) [reply src/Evolution/Executables/GrMhd/ValenciaDivClean/EvolveValenciaDivClean.hpp:72]: yes

**Review by nilsdeppe — APPROVED** [review]
@wthrowe could you please take a look at this when you get a chance? Thank you :)

**Comment by nilsdeppe** [comment]
Restarted GCC builds, clang tidy is a boost error we're ignoring for now.

**Comment by nilsdeppe** [comment]
Only clang-tidy is failing and we're ignoring that issue because it's inside Boost.

**Comment by nilsdeppe** [comment]
I can verify the input file behaves as expected :+1: Nice job @kidder! :smile:

**Review by wthrowe — APPROVED** [review]

**Pull request merged**